### DISCUSSION TITLE
batch1

### DISCUSSION THREAD

**Discussion by giscus** [thelondonschoolofthirukkural/thelondonschoolofthirukkural.github.io#3] [Announcements]
# batch1

http://localhost:7062/batch1.html

**Comment by naveenatmit** [comment]
For the class on 15/10/2023, 
the following words can be given importance:
---
* குணம்
* குற்றம்
---
* தீ புண்
* நா வடு
---
* முகம்
* அகம்
* நட்பு
---
* நாகாக்க
* சோகாக்க
---
